src/content/docs/mobile-monitoring/new-relic-mobile-ios/watchOS/watchOS-agent-setup.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Starting the New Relic agent to your watchOS project
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - Installation
+  - SwiftUI
+  - WatchOS
+metaDescription: How to properly setup your watchOS project with the New Relic agent.
+freshnessValidatedDate: never
+---
+
+WatchOS projects do not normally contain an `ApplicationDelegate` or `ExtensionDelegate` class. These delegates are necessary to configure and start the New Relic agent in a watchOS project. Before adding the AppDelegate, follow the [outlined procedure](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation) to correctly install the New Relic agent. 
+
+## WatchOS Application [#watchOS_App]
+
+To add an `AppDelegate` to your watchOS application, follow the steps:
+
+1. Add a new file to your project named `WatchAppDelegate.swift`
```

Comment:
Does it matter where in their project this file is added? Where should we recommend they put it?